### DISCUSSION TITLE
Tensorflow attribute issue when running LAMMPS with DeePMD model.

### DISCUSSION THREAD

**Discussion by kimiaGF** [deepmodeling/deepmd-kit#2779] [Q&A]
I recently built deepmd from source along with the GPU-enabled version of tensorflow, but now when I try to run a LAMMPS script on models that were trained with a condo-installed version of deepmd (GPU version), I get the following error:

``` INVALID_ARGUMENT: Tensor spin_attr/ntypes_spin:0, specified in either feed_devices or fetch_devices was not found in the Graph```

I re-froze the old model with the source-built deepmd executable but the error persists. Has anyone else encountered this?

**Comment by njzjz** [comment]
See #2749. It doesn't matter.
